### PR TITLE
Draft: getting-started: Add XMPP Providers and self-hosting possibility

### LINKED ISSUE #1083: XMPP Providers project on https://xmpp.org
[XMPP Providers](https://invent.kde.org/melvo/xmpp-providers) makes onboarding easier. Filtered provider lists are already [hosted on data.xmpp.net](https://github.com/xsf/infrastructure/issues/14).

Currently, there is only the mentioned project site. We think that an overview / welcome page on https://xmpp.org could reach more people. Thus, we suggest to create a page with the following content:

1. Introduction for everybody
2. Detailed section for users describing that they can use specific clients already integrating XMPP Providers or search providers on the web page (see point 4)
3. Short section for server admins / operators and client developers with a link to the project site
4. Table listing all providers automatically retrieved from the lists on data.xmpp.net (including filtering, sorting, searching, description why a provider is in a specific [category](https://invent.kde.org/melvo/xmpp-providers#categories) and how to improve that)
5. Form for submitting new providers as an alternative to directly creating a [merge request](https://invent.kde.org/melvo/xmpp-providers/-/merge_requests)

### PR DESCRIPTION
**Pull request by melvo**
Closes #1083

### CODE REVIEW

**Review by Echolon — CHANGES_REQUESTED** [review]
I would simply reorder and put providers at the top.

**Review by Echolon — APPROVED** [review]

**Comment by Echolon** [comment]
Deploy when website is ready.

**Review by Echolon — CHANGES_REQUESTED** [review]
Please lets only change the order

**Review by Echolon — APPROVED** [review]
Ok cool.

We can come back to this at a later stage. Im sure this will be the better path to go.

**Comment by Echolon** [comment]
@wurstsalat3000 merge if you are happy

File: content/getting-started.md [hunk 1]
```diff
@@ -34,13 +34,16 @@ There are [many, many XMPP clients](/software/clients) for you to choose from. T
...
 # 2. Create an account
 
-As with e-mail, you will need an account with a service provider. Again, there are plenty of choices. The following links provide lists of publicly available providers.
+As with email, you need an account with a service provider.
+Use a [public provider from the curated list of **XMPP Providers**](https://providers.xmpp.net) or be [your own one](https://xmpp.org/software/servers/).
 
-* [Public XMPP servers](https://list.jabber.at/), list.jabber.at
-* [Open list of public XMPP servers](https://xmpp-servers.404.city), xmpp-servers.404.city
-* You can also review the list of the [IM Observatory](https://xmpp.net/list.php) _(Service is eventually acting unexpected currently)_
+**XMPP Providers** combines different ratings and contains providers that are also in several uncurated lists.
```
Comment (wurstsalat3000) [inline content/getting-started.md:40]:
```suggestion
**XMPP Providers** combines various ratings to compose a curated list. There are several other lists of providers in the web, which might not be curated.
```

**Pull request merged**